benchmarks/utilities/log_parser.hpp

```diff
@@ -127,11 +129,25 @@ inline std::vector<event> parse_csv(std::string const& filename)
 {
   rapidcsv::Document csv(filename, rapidcsv::LabelParams(0, -1));
 
-  std::vector<std::size_t> tids     = csv.GetColumn<std::size_t>("Thread");
-  std::vector<std::string> actions  = csv.GetColumn<std::string>("Action");
-  std::vector<std::string> pointers = csv.GetColumn<std::string>("Pointer");
-  std::vector<std::size_t> sizes    = csv.GetColumn<std::size_t>("Size");
-  std::vector<uintptr_t> streams    = csv.GetColumn<uintptr_t>("Stream");
+  std::vector<std::size_t> tids    = csv.GetColumn<std::size_t>("Thread");
+  std::vector<std::string> actions = csv.GetColumn<std::string>("Action");
+
+  auto parse_pointer = [](std::string const& s, uintptr_t& ptr) {
+    ptr = std::stoll(s, nullptr, 16);
+  };
+
+  std::vector<uintptr_t> pointers = csv.GetColumn<uintptr_t>("Pointer", parse_pointer);
+  std::vector<std::size_t> sizes  = csv.GetColumn<std::size_t>("Size");
+
+  auto parse_stream = [](std::string const& s, rmm::cuda_stream_view& stream) {
+    cudaStream_t cs;
+    uintptr_t ls = std::stoll(s);
+    std::memcpy(&cs, &ls, sizeof(cudaStream_t));
+    stream = rmm::cuda_stream_view{cs};
+  };
```

Comment:
There's no need to reconstitute the contents of the log file as a `cudaStream_t` anymore than there is to reconstitute the thread ID as a `std::thread_id` or the pointer as a `void*`.